api/app.py

```diff
@@ -68,24 +93,9 @@ def analyze():
         response = requests.post(qujata_curl_target + "/curl", headers=headers, json=payload, timeout=request_timeout)
         # Print response details
         print('Status code:', response.status_code)
+        if(response.status_code < 200 or response.status_code > 299):
```

Comment:
I recommend to extract this line
response.status_code < 200 or response.status_code > 299
to a separate function